app/Kernel.php

```diff
@@ -71,6 +73,21 @@ public function boot()
     }
 
     /**
+     * We need to manually initialize SpoonSession for the Symfony sessions to work properly
+     */
+    private function fixSymfonySessions()
+    {
+        SpoonSession::start();
+
+        /** @var \Symfony\Component\HttpFoundation\Session\Session $session */
+        $session = $this->getContainer()->get('session');
```

Comment:
I would wrap this method still in a check where you check if the service exists, I think that in cases like a console command it doesnt exist, maybe in those cases also skipp strarting SpoonSession